sessionctx/variable/session.go

```diff
@@ -285,3 +285,32 @@ func (s *SessionVars) GetSystemVar(key string) types.Datum {
 	}
 	return d
 }
+
+// GetTiDBSystemVar get variable value for name.
+// The variable should be a TiDB specific system variable (The vars in tidbSysVars map).
+// If the session scope variable is not set, it will get global scope value and fill session scope value.
+func (s *SessionVars) GetTiDBSystemVar(ctx context.Context, name string) (string, error) {
+	key := strings.ToLower(name)
+	sVal, ok := s.systems[key]
```

Comment:
If the name is not TiDB specific, it still returns the value, do not confirm the method comment.